RonKbS/mydiary-react-app
# ##162190567 Setup CI toolS

### What does this PR do?
Enables creation of test and code coverage on raising a PR.

#### Description of Task to be completed?
Currently, there is no check that builds are successful for commits pushed to this repository.
This task enables automatic building using travis, as well as the display of test coverage using circleci and codeclimate using badges and checks on PRs raised.

#### How should this be manually tested?
On the `ch-setup-ci-tools-162190567` branch, a badge is displayed showing whether the build is passing or not.

####Snapshots
<img width="821" alt="screen shot 2018-11-29 at 17 54 06" src="https://user-images.githubusercontent.com/40039818/49230093-f2c4e480-f3ff-11e8-8022-3a56fe74c7cb.png">


#### What are the relevant pivotal tracker stories?
[162190567](https://www.pivotaltracker.com/story/show/162190567)
